com.zsmartsystems.zigbee.dongle.conbee/src/main/java/com/zsmartsystems/zigbee/dongle/conbee/ZigBeeDongleConBee.java

```diff
@@ -300,6 +300,7 @@ public void sendCommand(final int msgTag, final ZigBeeApsFrame apsFrame) {
 
         request.setRequestId(apsFrame.getApsCounter());
         request.setClusterId(apsFrame.getCluster());
+        request.setTxOptions(apsFrame.getAckRequest() ? 0x40 : 0);
```

Comment:
Please, can you test that the frame is indeed transmitted with APS Ack Request = FALSE as I don't have a dongle to test with.

Reply:
I don't know anyone using this coordinator